train_model.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+import h5py
+import numpy as np
+
+from keras.models import Sequential
+from keras.layers import Flatten, Dense, Lambda, Conv2D, Cropping2D, Dropout, MaxPooling2D
+from keras.optimizers import Adam
+from sklearn.model_selection import train_test_split
+
+from utils.resnet import resnet50_model
+from keras.backend import tensorflow_backend as backend
+
+
+# model name
+NVIDIA = "nvidia"
+RESNET50 = "resnet50"
+
+MODEL_NAME = NVIDIA
+# MODEL_NAME = RESNET50
+
+
+def option_parser() -> str:
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-s', '--source_training_data_path',
+                        required=True,

```

Comment:
必須にしてもいいと思うんだけど、毎回実行するときに指定するの大変かもしれないからdefaultにしてもいいかも。（あ、、でもまだどこに.h5ファイル置くか決めてないか...）

```suggestion
                        default='trainingData.h5'
```